lisp/ess-custom.el

```diff
@@ -549,12 +549,6 @@ ess-smart-comma only, but will be enriched in the near future.")
 Used to avoid annoying jumping by ess-eval.*-and-step to end of
 buffer or end chunks etc.")
 
-(defcustom ess-S-assign " <- "
-  "String used for left assignment in all S dialects.
```

Comment:
This was absoleted in favor of first entry in `ess-assign-list`? If so, the news should be explicit about this. It's extremely unlikely that users changed this variable, but who knows.

Reply:
It's in there already, I believe.

Reply:
I meant in the news, but if it's in the docs it's also good enough. 

BTW, could we make first element of ess-assing-list picking values from ess-S-assign in case that one was set by the user?

Reply:
Is this not what you're talking about? 

```
@item The option ess-S-assign has been removed.
Customize new option @code{ess-assign-list} instead.
```

Reply:
I thought it should be a bit more explicit about "customizing first element". Nevermind. Let's not complicate things.